src/XRP/xrp-client-decorator.ts

```diff
@@ -10,7 +10,7 @@ export interface XRPClientDecorator {
    * Retrieve the balance for the given address.
    *
    * @param address The X-Address to retrieve a balance for.
-   * @returns A `BigInteger` representing the number of drops of XRP in the account.
+   * @return A `BigInteger` representing the number of drops of XRP in the account.
```

Comment:
I would use `@returns` over `@return`.

Reasoning:
1. It reads more like a sentence ("Returns a BigInteger...")
2. [`@returns` is the standard for TSDoc](https://github.com/microsoft/tsdoc/blob/master/tsdoc/src/details/StandardTags.ts#L421), which does not yet support tag synonyms
3. [`@returns` is the one actually documented in JSDoc, with `@return` used as a potential synonym](https://jsdoc.app/tags-returns.html)

Reply:
great, I was wondering about this - thanks for clarifying.  Fixed them in the relevant files here, and made a note to myself to apply to the entire SDK for consistency.